src/cache/change-writer.js

```diff
@@ -49,6 +49,60 @@ class ChangeWriter {
       return JSON.parse(str)
     })
   }
+
```

Comment:
I'd actually intended change writer to be used purely as an app-level abstraction for writing API change logs. I think you should move these changes into a different file.

Reply:
Moved to [PersistentStore](83/files#diff-81abec4c30ce2f739a3bd69d706c3ef8)